lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -707,9 +734,29 @@ impl LayerModel {
         }
     }
 
-    /// Create a new sublayer to this layer, with the same camera.
+    /// Create a new sublayer to this layer. It will inherit this layer's camera.
     pub fn create_sublayer(&self, name: impl Into<String>) -> Layer {
-        let layer = Layer::new_with_cam(name, &self.camera.borrow());
+        let layer = Layer::new_with_flags(name, true, true);
+        self.add_sublayer(&layer);
+        layer
+    }
+
+    /// Create a new sublayer to this layer. Override the camera for this layer, breaking the camera
+    /// inheritance chain. Updates to the parent camera will not affect this layer.
+    pub fn create_sublayer_with_cam(&self, name: impl Into<String>, camera: &Camera2d) -> Layer {
+        let layer = Layer::new_with_cam(name, camera);
+        self.add_sublayer(&layer);
+        layer
```

Comment:
We are trying not to sue abbreviations in public APIs. This should be named `create_sublayer_with_camera` instead. We are OK with abbreviations in internal funcions or internal variables, but public ones should not use them.